tests/ShadowDOM/js/Document.js

```diff
@@ -230,14 +233,18 @@ htmlSuite('Document', function() {
 
     div.offsetHeight;
 
-    var list = document.querySelectorAll('div /deep/ aa');
```

Comment:
@addyosmani should I remove duplicate tests before squashing the commits  in this PR ?